### ISSUE TITLE
Got UFO Ending and didn't let me go back to title screen

### ISSUE THREAD

**Issue by Nessu4K** [elishacloud/Silent-Hill-2-Enhancements#1086]
A few things here, UFO Ending didn't end with the final note of the music, I had to turn off Fast Transitions to fix it because it transitioned too fast.
When I save the Results, the back button didn't leave the save screen and had to use the mouse to back out.
Which is a bit weird.

Also I have a question, under "Movies" the UFO Ending isn't unlocked. Does it not show up under Movies?

**Comment by Polymega** [comment]
> A few things here, UFO Ending didn't end with the final note of the music, I had to turn off Fast Transitions to fix it because it transitioned too fast.

I tested and didn't experience this issue.

> When I save the Results, the back button didn't leave the save screen and had to use the mouse to back out.
Which is a bit weird.

I also didn't experience this issue on keyboard. Where you using a controller?

> Also I have a question, under "Movies" the UFO Ending isn't unlocked. Does it not show up under Movies?

The UFO ending is not a movie (FMV), but rather a collection of still images. Therefor , it is not classified as a movie so it will not appear in the MOVIES menu.

**Comment by Nessu4K** [comment]
Hmm, interesting. The music thing happened to me and idk why. But I did find the fix of turning off Fast Transitions.

The back out results screen after saving it not working only did it one time, I tried another time and it didn't do it. You're right about the keyboard letting you back out. I only use controller to play this (Xbox) and this is the first time something like that happened.

Thanks for letting me know about the UFO Ending, I figured it was something like that. I got the rest of the endings under Movies, just need the Dog if it shows up there.
